projects/plugins/social/src/js/components/header/index.js

```diff
@@ -88,10 +90,22 @@ const Header = () => {
 								usedCount={ usedCount }
 								scheduledCount={ scheduledCount }
 								remainingCount={ remainingCount }
+								remainingLabel={ __( 'left in this cycle', 'jetpack-social' ) }
 								legendCaption={ __( 'Auto-share usage', 'jetpack-social' ) }
 								noticeType={ noticeType }
 								className={ styles[ 'bar-wrapper' ] }
 							/>
+							<Text variant="small" className={ styles[ 'bar-description' ] }>
+								{ createInterpolateElement(
+									__(
+										'<i>As a free Jetpack Social user, you get 30 shares within every rolling 30-day window.</i>',
+										'jetpack-social'
+									),
+									{
+										i: <i />,
+									}
+								) }
```

Comment:
There is no need to use `createInterpolateElement` here. We can simply use 

```jsx
<i>{ __( 'As a free Jetpack Social user, you get 30 shares within every rolling 30-day window.', 'jetpack-social' ) }</i>
```

Reply:
Fixed in 749967f, Thanks!